docs/log_collection.md

```diff
@@ -10,7 +10,7 @@ Send logs to Datadog from your Android applications with [Datadog's `dd-sdk-andr
 
 ## Setup
 
-1. Add the Gradle dependency by declaring the library as a dependency in your `build.gradle` file:
+1. Add the Gradle dependency by declaring the library as a dependency in your `build.gradle` file located in the root project directory:
```

Comment:
why this change is needed? this code block should be added to the particular module's `build.gradle`, not to the root (project) one.

Reply:
I can update to say the particular module's `build.gradle`. A customer gave feedback that they weren't sure which we were referencing.